### ISSUE TITLE
Released 0.13.0 artifacts do not correspond to the 0.13.0 commit

### ISSUE THREAD

**Issue by dhpiggott** [akka/akka-management#203]
I noticed that 0.13.0 includes https://github.com/akka/akka-management/pull/192 (nice simplification @StephenKing!):
```
➜  akka-management git:(master) git --no-pager show --no-patch --oneline v0.13.0
b878cd2 (tag: v0.13.0) Align management bind and bootstrap contact point (#192)
```
So I went ahead and prepared a small change to my project to make use of the simplification (removing `contact-point.fallback-port = 19999`). But my project's integration tests failed the change before I commited it, and inspection of the logs showed this:
```
zone-host_1     | INFO  akka.management.AkkaManagement AkkaManagement(akka://liquidity) - Binding Akka Management (HTTP) endpoint to: zone-host:19999
zone-host_1     | INFO  akka.management.AkkaManagement AkkaManagement(akka://liquidity) - Bound Akka Management (HTTP) endpoint to: zone-host:19999
```
So somehow 19999 was still creeping in from somewhere. `grep --recursive 19999` of my project's codebase was showing nothing, as it also did of https://github.com/akka/akka-management/tree/v0.13.0. So, time to check the Jar:
```
curl --silent https://dl.bintray.com/akka/maven/com/lightbend/akka/management/akka-management_2.12/0.13.0/akka-management_2.12-0.13.0.jar --output akka-management.jar
unzip akka-management.jar
grep 8558 reference.conf | wc --lines
grep 19999 reference.conf
```

I would expect 0.13.0 to include https://github.com/akka/akka-management/pull/192 given that the tag points to the commit that rebased it onto master. Am I mistaken in expecting this, or has the 0.13.0 release process gone wrong somehow?

**Comment by dhpiggott** [comment]
Thanks! I briefly tried figuring out how the release process works to see if that was the issue (before I just checked the Jar itself), and found myself wondering whether it would be worth using https://github.com/scalacenter/sbt-release-early?

**Comment by ktoso** [comment]
I released 0.13.1 which should be correct though I'm not 100% sure what could have gone wrong in the previous one... I did have trouble but not with the tag but getting it to publish hm...

https://search.maven.org/#search%7Cga%7C1%7Cakka-management
